credoai/modules/dataset_modules/dataset_fairness.py

```diff
@@ -64,16 +65,19 @@ def run(self):
         dict, nested
             Key: assessment category
             Values: detailed results associated with each category
-        """        
-        sensitive_feature_prediction_results = self._run_cv()
-        group_differences = self._group_differences()
-        normalized_mutual_information = self._calculate_mutual_information()
-        balance_metrics = self._assess_balance_metrics()
-        self.results = {**balance_metrics,
-                        **sensitive_feature_prediction_results,
-                        'standardized_group_diffs': group_differences,
-                        'normalized_mutual_information': normalized_mutual_information,
-                        }  
+        """
```

Comment:
This will lead to a nested `results` dictionary, which will not be displayed appropriately by the credo_reporter (see `display_results_tables`)

Reply:
I think we can do this by making the sensitive feature just a column with a label